src/shared/varlink-io.systemd.Resolve.Monitor.c

```diff
@@ -112,6 +112,30 @@ static SD_VARLINK_DEFINE_METHOD(
                 ResetStatistics,
                 VARLINK_DEFINE_POLKIT_INPUT);
 
+static SD_VARLINK_DEFINE_STRUCT_TYPE(
+                DNSServer,
+                SD_VARLINK_DEFINE_FIELD(address, SD_VARLINK_INT, SD_VARLINK_ARRAY),
+                SD_VARLINK_DEFINE_FIELD(addressFamily, SD_VARLINK_INT, 0),
+                SD_VARLINK_DEFINE_FIELD(port, SD_VARLINK_INT, 0),
+                SD_VARLINK_DEFINE_FIELD(interfaceSpecifier, SD_VARLINK_INT, SD_VARLINK_NULLABLE),
+                SD_VARLINK_DEFINE_FIELD(serverName, SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
+                SD_VARLINK_DEFINE_FIELD(accessible, SD_VARLINK_BOOL, 0));
```

Comment:
for new varlink IDL please always add full comments, i.e. use SD_VARLINK_FIELD_COMMENT(), SD_VARLINK_INTERFACE_COMMENT(), SD_VARLINK_SYMBOL_COMMENT().